.github/pull_request_template.md

```diff
@@ -0,0 +1,10 @@
+# **Problem:**
+
+
```

Comment:
Should we include a small bullet point list of what we are intending that the PR creator include in each section?

Reply:
Good idea. Thoughts on [these](https://github.com/Sage-Bionetworks-Workflows/py-dcqc/pull/58/commits/7e3958c72bcecd7f8f309f601083a9115d2adc77)?